.pre-commit-config.yaml

```diff
@@ -1,66 +1,57 @@
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.5.0
+    rev: v4.6.0
     hooks:
-      - id: no-commit-to-branch
+      - id: check-executables-have-shebangs
       - id: check-merge-conflict
+      - id: check-json
+        exclude: ^helmfile\.d/upstream/
+      - id: check-toml
+        exclude: ^helmfile\.d/upstream/
       - id: check-yaml
-        exclude: ^helmfile\.d/(bases/.*|charts/.*/templates/.*|stacks/.*|upstream/.*|state.yaml)$
+        exclude: ^helmfile\.d/.*/templates/|^helmfile\.d/upstream/|^helmfile\.d/state.yaml$
         args:
           - --allow-multiple-documents
-      - id: check-json
-        exclude: ^helmfile\.d/(charts/.*/templates/|upstream/)
       - id: detect-private-key
-        exclude: ^helmfile\.d/upstream/|^pipeline/sops-pgp-setup.bash$
+        exclude: ^helmfile\.d/upstream/
       - id: end-of-file-fixer
-        exclude: ^helmfile\.d/upstream/|^tests/.*/resources/
+        exclude: ^helmfile\.d/upstream/|^tests/.*/.*\.out$
+      - id: mixed-line-ending
+        exclude: ^helmfile\.d/upstream/
+      - id: no-commit-to-branch
+        exclude: ^helmfile\.d/upstream/
```

Comment:
Question: Is `no-commit-to-branch` the one that prevents us from committing to main? Is it intended to exclude `^helmfile\.d/upstream/` from that? If so, then what is the reasoning for that?

Reply:
Yeah, that's an error on my part.